tutorials/intermediate/tight_layout_guide.py

```diff
@@ -198,88 +198,15 @@ def example_plot(ax, fontsize=12):
 example_plot(ax1)
 example_plot(ax2)
 
-gs1.tight_layout(fig, rect=[0, 0, 0.5, 1])
+gs1.tight_layout(fig, rect=[0, 0, 0.5, 1.0])
 
 ###############################################################################
-# For example, this can be used for a figure with multiple gridspecs.
+# However, we do not recommend that this be used to manually construct more
+# complicated layouts, like having one GridSpec in the left and one in the
+# right side of the figure. For these use cases, one should instead take
+# advantage of :doc:`/gallery/subplots_axes_and_figures/gridspec_nested`, or
+# the :doc:`/users/next_whats_new/subfigures`.
```

Comment:
I assume there's a "better" way to reference "next_whats_new"?

Reply:
You should not reference whats_new files explicitly because the will be merged upon release. You could add a sphinx label if you really need to. However in general, features mentioned in whats_new should also be part of the regular documentation; in this case `gallery/subplots_axes_and_figures/subfigures`.

Reply:
So AFAICT there is no gallery entry for subfigures, I guess I should just add it? i was simply reticent to verbatim copy stuff from a "what's new" into a new gallery entry.